### DISCUSSION TITLE
Implement /player [botname] use perTick into normal carpet (from quickCarpet)

### DISCUSSION THREAD

**Discussion by bob-a-tea** [gnembon/fabric-carpet#945] [Ideas]
I was watching mango's 270k concrete converter video a few days ago, and I saw that he used quickCarpet to make bots place more than 72k blocks per hour (1 block per gt). Would it be possible to implement that feature (player [botname] use perTick) into normal carpet 1.16 and 1.17? Cheers!

**Comment by gnembon** [comment]
This sounds like a very specific feature request, so rather create a ticket in Issues this way it can be added. I see discussions for that are not to be immediately added.
